src/mame/apple/apple2video.h

```diff
@@ -71,8 +68,17 @@ class a2_video_device : public device_t, public device_palette_interface, public
 	void init_palette();
 
 private:
+	template <model Model, bool Invert, bool Flip>
+	void text_update(screen_device &screen, bitmap_ind16 &bitmap, const rectangle &cliprect, int beginrow, int endrow);
+
 	template <model Model, bool Invert, bool Flip>
 	void plot_text_character(bitmap_ind16 &bitmap, int xpos, int ypos, int xscale, uint32_t code, int fg, int bg);
+
+	void lores_update(screen_device &screen, bitmap_ind16 &bitmap, const rectangle &cliprect, int beginrow, int endrow);
+	void dlores_update(screen_device &screen, bitmap_ind16 &bitmap, const rectangle &cliprect, int beginrow, int endrow);
+	void dhgr_update(screen_device &screen, bitmap_ind16 &bitmap, const rectangle &cliprect, int beginrow, int endrow);
+
+	bool use_page_2();
```

Comment:
Wouldn’t it make sense for this to be `const` qualified?

Reply:
I think a lot of the methods in this class could be `const`, actually.